src/pages/waiting-room-to-car/waiting-room-to-car.analytics.effects.ts

```diff
@@ -29,10 +42,45 @@ export class WaitingRoomToCarAnalyticsEffects {
   @Effect()
   waitingRoomToCarViewDidEnter$ = this.actions$.pipe(
     ofType(WAITING_ROOM_TO_CAR_VIEW_DID_ENTER),
-    switchMap((action: WaitingRoomToCarViewDidEnter) => {
+    withLatestFrom(
+      this.store$.pipe(
+        select(getTests),
+        select(getCurrentTestSlotId),
+      ),
+      this.store$.pipe(
+        select(getTests),
+        select(getCurrentTest),
+        select(getJournalData),
+        select(getCandidate),
+        select(getCandidateId),
+        ),
+    ),
+    switchMap(([action, slotId, candidateId]: [WaitingRoomToCarViewDidEnter, string, number]) => {
+      this.analytics.addCustomDimension(AnalyticsDimensionIndices.CANDIDATE_ID, `${candidateId}`);
```

Comment:
Interesting that the candidate ID is a desirable custom dimension here. From the standpoint of using analytics to identify usability issues, why does it matter which candidate the examiner attempted to submit form validation errors for?

Reply:
The candidate ID and test slot ID can be used to tie the whole test journey together and get stats on screen usage / time. I didn't think the errors had the extra dimensions attached to them?